apiserver/facades/client/bundle/bundle.go

```diff
@@ -160,6 +216,6 @@ func (b *BundleAPI) ExportBundle() (params.StringResult, error) {
 	return params.StringResult{}, nil
 }
 
-// Mask the new method from V1 API.
 // ExportBundle is not in V1 API.
-func (u *APIv1) ExportBundle() (_, _ struct{}) { return }
+// Mask the new method from V1 API.
```

Comment:
I am not sure how this got in... We do not generally implement versions this way but rather using interfaces as in https://github.com/juju/juju/blob/develop/apiserver/facades/client/modelmanager/modelmanager.go#L44

Reply:
This is existing code from develop. I just changed the doc to make linter happy

Reply:
This looks wrong anyway. The version bump need to be changed to avoid doing this kind of placeholder, I think.

But even if you choose not to undertake this here, the comment should be starting with the method name... Maybe remove this change?